app/src/main/java/com/hyperion/ths/marvel_03/ui/OnItemClickListener.java

```diff
@@ -0,0 +1,15 @@
+package com.hyperion.ths.marvel_03.ui;
+
+/**
+ * Created by ths on 07/06/2017.
+ */
+
+import com.hyperion.ths.marvel_03.data.model.Hero;
+import com.hyperion.ths.marvel_03.ui.BaseRecyclerView;
```

Comment:
WARNING: [ImportFromSamePackage] No need to import a type that lives in the same package
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#ImportFromSamePackage